src/support_sphere/lib/presentation/components/auth/signup_form.dart

```diff
@@ -89,11 +116,21 @@ class _SignupCodeInput extends StatelessWidget {
         return TextFormField(
           enabled: !state.status.isInProgress,
           key: const Key('signupForm_couponCodeInput_textFormField'),
-          onChanged: (value) => context.read<SignupCubit>().signupCodeChanged(value),
+          onChanged: (value) =>
+              context.read<SignupCubit>().signupCodeChanged(value),
+          autovalidateMode: AutovalidateMode.onUserInteraction,
+          validator: (value) => validateValue(
+            [
+              FormBuilderValidators.required(),
+              FormBuilderValidators.equalLength(7),
```

Comment:
Is this the right length? Seems inconsistent with the currently hardcoded codes: https://github.com/uw-ssec/post-disaster-comms/blob/7fdbd57d0726e9a8866777013ad52dc574860fc1/src/support_sphere/lib/data/services/auth_service.dart#L7-L11

Reply:
Ooops, it used to be 6, and I've decided to be 7 to match a short git sha type. I'll update that in a different PR

Reply:
Will be updated at https://github.com/uw-ssec/post-disaster-comms/pull/89/files#diff-28eec2ca3392e24ba46c6fb8e19cd71233c43de6bba792c24085a35fc1e20caf